charts/headlamp/values.schema.json

```diff
@@ -35,7 +35,13 @@
       "type": "array",
       "description": "Image pull secrets",
       "items": {
-        "type": "string"
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string",
+            "description": "ImagePullSecrets is an optional list of references to secrets in the same namespace to use for pulling any of the images used by this PodSpec. If specified, these secrets will be passed to individual puller implementations for them to use."
```

Comment:
This is the name field description, but the it reads like it is for the parent. I think this should be moved up.

Reply:
Oops, my bad. Fixed. Thanks again :-)